en/admin/config.rst

```diff
@@ -566,6 +570,63 @@ The following are parameters related to the database server. The type and value
   
     **tcp_keepalive** is a parameter which specifies if you apply SO_KEEPALIVE option to TCP network protocol or not. The default is **yes**. If this value is **no**, DB server-side connection can be disconnected when transaction logs are not copied for a long time in the firewall environment between master and slave.
  
+
+**use_user_hosts** 
+
+    **use_user_hosts** is a system paramter to use to select the host look up between **hostname** and **IP address**, required by the CUBRID service from the services below. The default value is **OFF**.
```

Comment:
Here are the changes that should be made: 
- "...is a system parameter that is used to select the host look up..."
- paramter --> parameter